andino_apps/README.md

```diff
@@ -0,0 +1,24 @@
+# Andino Apps
+
+This package contains more complex launh files that execute applications with the andino robot.
+
+# Usage
+
+## Gazebo classic simulation + Nav2
+
+A launch file for running the andino_gz_classic simulation and the nav2 stack is provided.
+It uses the [turtlebot3_world](https://github.com/ROBOTIS-GIT/turtlebot3_simulations/tree/master) world (_Apache 2 license_) by default.
+
+```
+ ros2 launch andino_apps andino_simulation_navigation.launch.py
+```
+
+To test the navigation inside rviz:
+
+- click in 2D pose estimate button and select the initial pose of the robot
+- click in nav2 Goal button and select the final point.
+- the robot will start to move to the selected goal.
```

Comment:
Probably we can add a GIF showing how to launch RViz and performing these operations. An image is worth 1000 words, you can imagine a video!

Reply:
Noted that!